server/src/test/java/org/elasticsearch/index/IndexModuleTests.java

```diff
@@ -718,6 +730,79 @@ public void onIndexCommitDelete(ShardId shardId, IndexCommit deletedCommit) {
         }
     }
 
+    public void testCustomRecoverySourceFactory() throws IOException {
```

Comment:
I think you mean ReplicationTrackerFactory?

```suggestion
    public void testCustomReplicationTrackerFactory() throws IOException {
```